### PR TITLE
Update alarm_control_panel.egardia.markdown

### PR DESCRIPTION
**Pull request by jeroenterheerdt**
**Description:**
Updating docs reflecting changes in egardia component no longer requiring separate egardiaserver install.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11344
## Checklist:

  - X ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/alarm_control_panel.egardia.markdown [hunk 1]
```diff
@@ -32,17 +32,18 @@ Configuration variables:
...
+2. Once logged in, go to *System Settings*, *Report* and change the Server Address for your primary server to the IP or hostname of your Home Assistant machine. You can leave the port number set to 52010 or change it to anything you like. **Make sure to change the settings of the primary server otherwise the messages will not come through. Note that this will limit (or fully stop) the number of alarm messages you will get through Egardia's / Woonveilig services.** Maybe, that is just what you want, because it removes the need to pay for their subscription. Make sure to save your settings by selecting 'OK'.
```
Comment (MartinHjelmare) [inline source/_components/alarm_control_panel.egardia.markdown]: I'm not sure if we should mention the subscription. Maybe we should leave it out to stay neutral?

Comment (jeroenterheerdt) [reply source/_components/alarm_control_panel.egardia.markdown]: it's kind of the whole purpose of the component, but sure, I am fine with leaving it out. I expect people to be able to read between the lines.

**Comment by jeroenterheerdt** [comment]
I removed the mention of the subscription.

**Comment by jeroenterheerdt** [comment]
Should this not be on the current branch since it is docs for an existing component? I thought the next branch was only for new components and current for updates for docs for existing components..

**Comment by MartinHjelmare** [comment]
Since the PR with the code changes will go in the next release, ie 0.61, this PR should target `next` branch. If this PR would have contained fixes of the description of current behavior it should have targeted `current` branch.

**Comment by jeroenterheerdt** [comment]
@MartinHjelmare  then the docs on the HA site are not entirely in line with this. Have a look at the [documentation on docs](https://home-assistant.io/developers/documentation/): 

> Create a Pull Request (PR) against the next branch of home-assistant.github.io if your documentation is a new feature, platform, or component.
Create a Pull Request (PR) against the current branch of home-assistant.github.io if you fix stuff, create Cookbook entries, or expand existing documentation.

**Comment by MartinHjelmare** [comment]
A fix to the documentation, would be if something is wrong with the documentation and no code has changed. Since you have changed code, you're not fixing the documentation, you're updating it to match the added code.

**Pull request merged**